Gems/Atom/RHI/DX12/Code/Source/RHI/StreamingImagePool.cpp

```diff
@@ -12,13 +12,23 @@
 #include <RHI/Image.h>
 #include <RHI/ResourcePoolResolver.h>
 #include <Atom/RHI/MemoryStatisticsBuilder.h>
-// NOTE: Tiled resources are currently disabled, because RenderDoc does not support them.
-// #define AZ_RHI_USE_TILED_RESOURCES
+
+// enable tiled resource implementation
+#define AZ_RHI_USE_TILED_RESOURCES
```

Comment:
We can probably move this to RHI level. Other backend ends can decide to use it or not.

Reply:
I may move it when we have other backends start to support it. such as Vulkan